openmdao/core/group.py

```diff
@@ -941,7 +951,9 @@ def solve_linear(self, dumat, drmat, vois, mode=None, solver=None):
         if len(rhs_buf) == 0:
             return
 
+        solver.rel_inputs = rel_inputs
         sol_buf = solver.solve(rhs_buf, self, mode=mode)
+        solver.rel_inputs = None
```

Comment:
curious why rel_inputs is not a (keyword) arg to the function call...  this looks kind of fishy.